docs/diagrams/DeleteMultiIndexActivityDiagram.puml

```diff
@@ -1,16 +1,22 @@
 @startuml
+skinparam defaultTextAlignment center
 
 start
-:User requests to delete multiple indexes;
-:Check validity of each index;
-
-if () then ([all indexes valid])
-    :Delete cases associated with each index;
-    :Display success message;
-    stop
+:User requests to\ndelete multiple indexes;
+:Check that indexes and dates\nhave not been used together;
+if () then ([indexes and dates not mixed])
```

Comment:
```suggestion
if () then ([indices and dates not mixed])
```

Reply:
TT both "indexes" and "indices" are accepted plural forms of "index"! with "indices" typically being used in more technical contexts.

We've used "indexes" throughout the UG and DG.